applications_tests/gd_navier_stokes_2d/taylor-green-vortex_gd_bdf1.prm

```diff
@@ -10,7 +10,7 @@ subsection simulation control
   set time step               = 0.1    # Time step
     set number mesh adapt     = 0       # If steady, nb mesh adaptation
   set time end                = 1.00      # End time of simulation
-  set output path             =  # Path to where the output results are made
+  set output path             = ./ # Path to where the output results are made
```

Comment:
It would be better to put the default value of output path to ./ and just remove this line from the prm file!
At the same time, i would take the opportunity to remove the output of these tests by putting output frequency to 0.

Reply:
the default value of the output path is already ./ so I removed all the set output path lines from the parameter handler files. The second comment is also applied.